CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Enhancements
 
+- Add a ci workflow for release automation
+  (PR[#4479](https://github.com/scality/metalk8s/pull/4479))
```

Comment:
There is no need to mention it in the changelog to me, it's not a change on the product it self

Reply:
Dropped!